pg_service_parser/ui/pg_service_dialog.ui

```diff
@@ -326,6 +326,143 @@
        </item>
       </layout>
      </widget>
+     <widget class="QWidget" name="tabConnections">
+      <attribute name="title">
+       <string>Connections</string>
```

Comment:
QGIS Connections

Reply:
Done in https://github.com/opengisch/qgis-pg-service-parser-plugin/pull/39/commits/a0468a33be83f9f5c3f5053eb10e4ea9b32bc6cb